include/ddc/discrete_domain.hpp

```diff
@@ -174,6 +174,12 @@ class DiscreteDomain
         return DiscreteDomain(front() + n1, extents() - n1 - n2);
     }
 
+    KOKKOS_FUNCTION constexpr DiscreteElement<DDims...> operator()(
```

Comment:
This looks like an essential operator. Till now, what did the user do to achieve same functionality?

Reply:
This operation used to be done directly in `Chunk` and `ChunkSpan`, see https://github.com/CExA-project/ddc/blob/9f53744b90f0b216fa27217561a4d9b21344b6d1/include/ddc/chunk_span.hpp#L338